docs/components/rpa/production.md

```diff
@@ -21,12 +29,18 @@ In the RPA script, your secrets are stored in the `${secrets}` variable. You can
 If you manage multiple RPA worker machines and scripts, you might need specialized environments to run certain tasks.
 To differentiate different capabilities of runners, you can use tags and labels.
 
+Labels are set both on the RPA task in the diagram and on your worker. To ensure your script is only executed by machines with the correct capabilities for this script, we recommend adding labels to your worker.
+
 In the `rpa-worker.properties`, add `camunda.rpa.zeebe.worker-tags=accounting-system`. This worker will now only take up jobs
 which are labeled `accounting-system`. If you also want the worker to work on unlabeled tasks, use `camunda.rpa.zeebe.worker-tags=default,accounting-system` instead.
 Each worker can have multiple labels and will pick up waiting jobs from all scripts.
 
 You can add labels to your script when configuring the RPA task in your diagram. Note that a script can only have a single label.
 
+Labels describe capabilities. If you want your worker to only pick up a specific script, you will need to use a unique label on both the worker and the RPA task. A worker can have multiple labels and will pick up any script that matches one of the given tags. For example, your worker might have access to the SAP application, but if you also want it to pick up browser automation tasks, add `SAP,BROWSER_AUTOMATION` to your worker tags. This will pick up tasks tagged as `SAP` and tasks tagged as `BROWSER_AUTOMATION`.
```

Comment:
SAP,BROWSER_AUTOMATION

This is intentionally a comma, yes? @marstamm

Reply:
Yes. On the Worker, we have a comma-separated list of Labels